CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 
-- …
+## [0.0.1] - 2020-09-29
```

Comment:
We've generally tagged our releases as `v0.0.1`

Reply:
Removed the version, as we can argue that this is not yet a release.